PendingReleaseNotes.md

```diff
@@ -5,3 +5,5 @@
 - Removed support for Ceph Pacific (v16)
 
 ## Features
+
+- Added `pgHealthyRegex` field to DisruptionManagementSpec. `pgHealthyRegex` is a regular expression that is used to determine which PG states should be considered healthy.
```

Comment:
Do you want to backport this feature to the 1.12 branch? This PendingReleaseNotes.md should only be updated if it is a feature only for 1.13.

Reply:
My team intended to backport this PR to v12. I added backport-release-1.12 label.

Reply:
Sounds good, then please revert this change to the pending release notes. It's only for 1.13 features.

Reply:
Thank you for letting me know. I fixed this and force pushed.